app/assets/stylesheets/alchemy/image_library.scss

```diff
@@ -0,0 +1,137 @@
+.picture-detail-bottom {
+  @include box-sizing(border-box);
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 210px;
+  background-color: $medium-gray;
+  padding-top: 2*$default-padding;
+  @include box-shadow(0 -2px $default-padding -2px $text-color);
+
+  .picture-detail-form {
+    position: relative;
+    left: -75px;
+    @include clearfix;
+    max-width: 1000px;
+    margin: 0 auto;
+  }
+
+  form, .picture-detail-usage {
+    @include box-sizing(border-box);
+    float: left;
+    width: 50%;
+    padding: 0 8px;
+  }
+}
+
+#zoomed_picture_background {
+  @include box-sizing(border-box);
+  width: 100%;
+  height: 100%;
+  padding-top: 33px;
+  padding-bottom: 210px;
+  margin: 0 auto;
+  text-align: center;
+  cursor: pointer;
+
+  &:before {
+    content: '';
+    vertical-align: bottom;
+    display: inline-block;
+    height: 100%;
+    margin-left: -4px;
+  }
+
+  .spinner {
+    position: absolute !important;
+    left: 50% !important;
+    top: 50% !important;
+    @include transform(translateXY(-50%));
+  }
+
+  img {
+    display: inline-block;
+    height: auto;
+    max-width: 100%;
+    max-height: 100%;
+    @include box-shadow(0 0 2*$default-margin $text-color);
+    background-color: $dark-gray;
+    vertical-align: bottom;
+    cursor: default;
+  }
+}
+
+.alchemy-image-overlay {
+
+  &.open {
+    background-color: rgba(0, 0, 0, 0.6);
+  }
+}
+
+.alchemy-image-overlay-container,
+.alchemy-image-overlay-dialog {
+  width: 100%;
+  height: 100%;
+  max-height: 100%;
+  max-width: 100%;
+}
+
+.alchemy-image-overlay-body {
+  width: 100%;
+  height: 100%;
+
+  h1 {
+    position: absolute;
```

Comment:
Properties should be ordered color, font-size, left, margin, padding, position, text-align, top, width